openhtf/plugs/usb/__init__.py

```diff
@@ -36,56 +38,153 @@ def MyPhase(test, adb):
 conf.Declare('usb_server', 'USB Server IP/Hostname')
 conf.Declare('usb_server_port', 'USB Server Port',
              default_value=10000)
-
 conf.Declare('libusb_rsa_key', 'A private key file for use by libusb auth.')
+conf.Declare('device_type', 'android, fastboot or general usb type')
```

Comment:
hm, it's unclear what this means... which device's type are you specifying?  i don't think this is a config thing, i think when you want an android device, you use an android plug, and when you want a fastboot device, you use a fastboot plug...